pnpm-lock.yaml

```diff
@@ -176,13 +176,20 @@ importers:
         version: 0.1.6(@types/react@18.0.28)(react-dom@18.2.0)(react@18.2.0)
       next:
         specifier: ^13.0.6
-        version: 13.2.1(@babel/core@7.21.4)(react-dom@18.2.0)(react@18.2.0)
+        version: 13.2.1(@babel/core@7.21.3)(react-dom@18.2.0)(react@18.2.0)
       nextra:
         specifier: latest
+<<<<<<< HEAD
```

Comment:
this lockfile is broken... this merge is broken

Reply:
Done